clarity/src/vm/costs/mod.rs

```diff
@@ -46,6 +45,10 @@ type Result<T> = std::result::Result<T, CostErrors>;
 
 pub const CLARITY_MEMORY_LIMIT: u64 = 100 * 1000 * 1000;
 
+// TODO: factor out into a boot lib?
```

Comment:
This is probably the right thing to do, if possible: create a "boot" crate that has a top-level trait that gets invoked at bootup time and then once per block thereafter, and will (in the former case) instantiate `.pox`, `.costs`, `.bns`, etc. and fill in the initial genesis balances as well as (in the latter case) check for the appropriate block heights at which to deploy new system boot code like `.costs-2` and (eventually) `.pox-2`.  Subnets and appchains can have their own boot implementations this way, and `libclarity` doesn't need to be peppered with bootup code.

Reply:
Is this going to be addressed in this PR?  If not, can you create an issue for it?

Reply:
No, I don't think so, so I'll open an issue for it.